lnrpc/chainrpc/chainkit.proto

```diff
@@ -12,6 +12,11 @@ service ChainKit {
     */
     rpc GetBlock (GetBlockRequest) returns (GetBlockResponse);
 
+    /*
```

Comment:
Since there is a CLI command now, can add the
```
lncli: `chain getblockheader`
```
annotation for the API docs generator.